fairscale/optim/grad_scaler.py

```diff
@@ -31,9 +32,16 @@ class ShardedGradScaler(TorchGradScaler):
 
     def __init__(self) -> None:
         super().__init__()
+        self.warning = True
```

Comment:
change to "warned"?

Reply:
it's "not warned" right now, renaming to "display_warning" if it's ok for you, to be super explicit ?